.drone.star

```diff
@@ -1,5 +1,14 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.12.0-rc2",
+            "qa": "https://download.owncloud.com/server/testing/owncloud-complete-20230228-qa.tar.bz2",
+            "tarball": "https://download.owncloud.com/server/stable/owncloud-complete-20230228.tar.bz2",
+            "tarball_sha": "86a9895f22be820eef7c7434cadea1b7476c8e5416fec902526d9a626a0c85c4",
+            "php": "7.4",
+            "base": "v20.04",
+            "tags": [],
```

Comment:
Don't know if there is a fallback to `value` but I would recommend setting the tags list to `"tags": ["10.12.0-rc2"]`.

Reply:
Same here. The instruction says empyt tags for RC. I'll change that too. Thanks for the hints.